src/Options/index.js

```diff
@@ -194,6 +194,9 @@ export interface ParseServerOptions {
   /* Session duration, in seconds, defaults to 1 year
   :DEFAULT: 31536000 */
   sessionLength: ?number;
+  /* Default limit for the size of results set on queries, defaults to `100`.
```

Comment:
```suggestion
  /* Default value for limit option on queries, defaults to `100`.
```

Same wording as `maxLimit` option, to make it more intuitive that it describes a related parameter.